aptos-move/framework/aptos-framework/sources/object.move

```diff
@@ -47,6 +48,9 @@ module aptos_framework::object {
     /// Explicitly separate the GUID space between Object and Account to prevent accidental overlap.
     const INIT_GUID_CREATION_NUM: u64 = 0x4000000000000;
 
+    /// generate_unique_address uses this for domain separation within its native implementation
+    const DERIVE_AUID_ADDRESS_SCHEME: u8 = 0xFB;
```

Comment:
move it below MAXIMUM_OBJECT_NESTING, so it is right before 0xFC constant

Reply:
Done.